__tests__/app.test.js

```diff
@@ -87,6 +87,43 @@ describe("GET /api/articles", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status 200, should return array of comments for an article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(11);
+        expect(comments[0].created_at).toBe("2020-11-03T21:00:00.000Z");
```

Comment:
I like the intention here - I think just checking one item might be a little undercooked. I recommend having a look at using [jest-sorted](https://www.npmjs.com/package/jest-sorted) as a tool to help you do this neatly!